listenbrainz/db/user_timeline_event.py

```diff
@@ -212,17 +208,18 @@ def hide_user_timeline_event(user_id: int, event_type: UserTimelineEventType, ev
         raise DatabaseException(str(e))
 
 
-def get_hidden_timeline_events(user: int, count: int) -> List[HiddenUserTimelineEvent]:
+def get_hidden_timeline_event(user_id: int, count: int) -> List[HiddenUserTimelineEvent]:
```

Comment:
Inversely to my original comment, the docline says "Retrieves all events…" so here the  plural name `get_hidden_timeline_events` was appropriate